software-templates/techdocs-template/skeleton/README.md

```diff
@@ -0,0 +1,63 @@
+[![Package Status](https://${{values.destination.host}}/${{values.destination.owner}}/${{values.destination.repo}}/actions/workflows/publish.yaml/badge.svg)](https://${{values.destination.host}}/${{values.destination.owner}}/${{values.destination.repo}}/actions/workflows/publish.yaml)
+
+
+# ${{values.name}}
+
+${{values.description}}
+
+## About
+
+This is a default [README.md](README.md) file generated by the template. Customize it for your project.
+
+Refer to the [index.md](docs/index.md) file for instructions on how to add documentation.
+
+
+## Setup
+
+Replace this section with instructions to set up your project locally. 
```

Comment:
is the MD working without the space here?

Reply:
@jenreiher-bcgov - I'm not sure which space? Is it the space between the `##` and the word `Setup` or the line break between `## Setup` and the sentence `Replace this section...`

Do you want me to remove the space?

Reply:
Oh, I see it now. I will add in a space